WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -345,6 +357,26 @@ private extension ProductsViewController {
         navigationItem.rightBarButtonItems = rightBarButtonItems
     }
 
+    func configureNavigationBarForEditing() {
+        configureNavigationBarTitleForEditing()
+        configureNavigationBarRightButtonItemsForEditing()
+    }
+
+    func configureNavigationBarTitleForEditing() {
+        let selectedProducts = tableView.indexPathsForSelectedRows?.count ?? 0
```

Comment:
I'm not sure if we can use `tableView.indexPathsForSelectedRows` safely. When the table view reloads due to a page being loaded this property gets cleaned.


https://user-images.githubusercontent.com/562080/210893792-06fcf688-9cd1-42fa-87ad-4d90bf348b04.mov

I think we will have to change how we update the table view with new pages or we will have to store these selections on a view model.
 
What do you think?

Reply:
Good catch! Added `ProductsListViewModel` to store selection state.